robocop/config.py

```diff
@@ -245,32 +250,40 @@ def _create_parser(self):
         optional.add_argument('-h', '--help', action='help', help=self.HELP_MSGS['help_info'])
         optional.add_argument('-v', '--version', action='version', version=__version__,
                               help=self.HELP_MSGS['help_version'])
+        optional.add_argument('--verbose', action='store_true', help=self.HELP_MSGS['verbose'])
         optional.add_argument('--directives', action='version', version=self.HELP_MSGS['directives'],
                               help=argparse.SUPPRESS)
 
         return parser
 
     def parse_opts(self, args=None, from_cli=True):
         args = self.preparse(args) if from_cli else None
-        if not args:
-            args = self.load_default_config_file()
-        if not args:
+        if not args or args == ['--verbose']:
+            loaded_args = self.load_default_config_file()
+            if loaded_args is not None:
+                # thanks for this we can have config file together with some cli options like --verbose
+                args = [*args, *loaded_args] if args is not None else loaded_args
+        if not args or args == ['--verbose']:
```

Comment:
This is a duplicate of the condition above

Reply:
It's not - args can be empty in the line 261 (no cli arguments, or only --verbose) but then there could be .robocop file with arguments. Depending if we were able to find .robocop file, args could be still empty or not. That's why we need to check for it again before looking for pyproject.toml

Reply:
But when I think about it, I think load.pyproject_file should be called only if loaded_args is None - we will avoid the situation where there .robocop with only --verbose inside.